### PR TITLE
ja-trans: add docs/_index.md

### PR DESCRIPTION
**Pull request by xichengliudui**
jo-trans: add docs/_index.md

### CODE REVIEW

File: content/ja/docs/_index.md [hunk 1]
```diff
@@ -0,0 +1,9 @@
+<!--
+---
+title: Documentation
+---
+-->
+
+---
```
[the remark this answers is not in the document]
Comment (xichengliudui) [reply content/ja/docs/_index.md]: I modified it, do you think it is ok?

File: content/ja/docs/_index.md [hunk 2]
```diff
@@ -0,0 +1,9 @@
...
+title: ファイル
```
Comment (cstoku) [inline content/ja/docs/_index.md]: How about making the translation of 'Documentation' to 'ドキュメント'?

File: content/ja/docs/_index.md [hunk 3]
```diff
@@ -0,0 +1,9 @@
...
+---
+
+<!--
```
Comment (cstoku) [inline content/ja/docs/_index.md]: You can either delete the comment.

**Comment by xichengliudui** [comment]
@cstoku  Is that ok?

File: content/ja/docs/_index.md (hunk 2 above)
Comment (cstoku) [reply content/ja/docs/_index.md]: @xichengliudui How about this?

Comment (xichengliudui) [reply content/ja/docs/_index.md]: @cstoku  Good good

**Comment by cstoku** [comment]
@xichengliudui Sorry, I have one more thing to ask.
Is there any reason for PR to master branch?

**Comment by xichengliudui** [comment]
why hold？

**Comment by xichengliudui** [comment]
I commit based on the main branch. Do I commit based on other branches?

**Comment by cstoku** [comment]
Currently the Japanese translation team is translating based on version 1.12 (dev-1.12.1 branch).
Could you start with `dev-1.12.1` branch and recreate PR that merge into `dev-1.12.1` branch?

**Pull request closed without merging**